CRM/Contact/Form/Task/PDFTrait.php

```diff
@@ -239,15 +239,10 @@ public function postProcess(): void {
     }
 
     foreach ($form->_contactIds as $item => $contactId) {
-      $caseId = $form->getVar('_caseId');
-      if (empty($caseId) && !empty($form->_caseIds[$item])) {
```

Comment:
why item ^^

Reply:
Off the top of my head maybe as a way to preserve order when there's multiple contact/case? So that you get the corresponding case for that contact? Something like that came up a year or two ago.